index.js

```diff
@@ -7,9 +7,16 @@
 var fs = require('fs');
 var path = require('path');
 var pathSep = require('path').sep;
+const crypto = require("crypto");
```

Comment:
I didn't find it in package.json. Also, do you believe it would be possible to separate all the crypto login in another file that only would be loaded in the case the cryptography was activated?

Reply:
I believe crypto is native to node like "fs". I assume this is why Flo wanted to go with it [here](https://github.com/parse-community/parse-server-fs-adapter/pull/8#issuecomment-307992864) 

> I really appreciate the effort on this PR, and that seems highly valuable. Could we use the native crypto libraries instead of node-cipher as it doesn't look heavily supported or popular and we try to limit dependencies over 3rd party libraries when possible.

For example, I never had to add it the package.json to use it and neither does the testcases

Reply:
Yes. I've just checked here. Sorry for my mistake.